contents/javascript/Javascript-framework-Angular.md

```diff
@@ -0,0 +1,202 @@
+<frontmatter>
+  title: Introduction to Angular
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# Introduction to Angular
+
+Author(s): [Ronak Lakhotia](https://github.com/RonakLakhotia)
+
+## What Is Angular?
+
+Angular is an open-source frontend Javascript framework that helps you build <tooltip content="Web apps that are more responsive to user interactions ">Single Page Web Applications</tooltip>.
+The framework is designed to better engineer applications in an effort to make development and testing easier for enterprise web applications. Angular is maintained by Google, as well as a community of individual developers.
+
+>Angular started as a side project. Back in 2009, Miško Hevery and Adam Abrons released a project that would help developers, as well as designers, build web applications using simple HTML tags. The name “Angular” comes from the angle brackets, or < >, that surround all HTML tags.
+>
+>Miško described the idea behind the framework in an <a href="https://www.infoworld.com/article/2612801/javascript/whats-so-special-about-googles-angularjs.html">interview done in 2013</a>. 
+
+
+## Why learn Angular?
+
+Angular is not the world’s simplest framework, and it takes time to understand some of the concepts that Angular has to offer.
+But once you do, Angular makes it easier to create rich and interactive user interfaces. 
+
+1. **Angular supports Single Page Applications.** Single Page Applications are a type of web applications that load a single HTML page, and the page is updated dynamically according to the interaction of the user. SPAs can communicate with the back-end servers without refreshing the full web page, for loading data in the application. They provide a better user experience.
+The fact that Angular supports the development of SPAs, is a good reason to learn the framework.
+ 
+2. **Angular uses TypeScript.** Angular applications are built using TypeScript language, a superscript of JavaScript. It helps catch errors early when writing the code or performing maintenance tasks. This language improves security by providing services like navigation, refactoring, and autocompletion thereby easing the development process for engineers. You can also opt out of its inbuilt features when needed.
```

Comment:
> This language improves security by providing services like navigation, refactoring, and autocompletion thereby easing the development process for engineers.

The mentioned sounds more like productivity features rather than security features. Futhermore, you also mentioned `easing the development process for engineers`, which aligns towards productivity.